icaruscode/CRT/crttpctrutheff_mc.fcl

```diff
@@ -54,31 +54,32 @@ outputs:
     fastCloning: false
   }
 }
-
 # The 'physics' section defines and configures some modules to do work on each event.
 physics:
 {
    analyzers:
   {
-    CRTTPCTruthEff:
+    CRTTPCTruthEffMC:
     {
-      module_type:         "icaruscode/CRT/CRTTPCTruthEff"
+      module_type:         "icaruscode/CRT/CRTTPCTruthEffMC"
       # The input parameters	   
       CRTHitLabel:         "crthit"           # CRT producer module label
       TPCTrackLabel:       ["pandoraTrackGausCryoE", "pandoraTrackGausCryoW"]     # Track producer module label
       PFParticleLabel: 	   ["pandoraGausCryoE", "pandoraGausCryoW"]
+      PFParticleMetadataLabel: ["pandoraGausCryoE", "pandoraGausCryoW"]
       #TPCTrackLabel:       "pandoraTrackGausCryoE"     # Track producer module label  
       TriggerLabel:        "daqTrigger"
```

Comment:
For MC, I find that I need to change "daqTrigger" to "emuTrigger." Unless there's been some change to this since v09_72ish, I would expect the same behavior here.